public/js/index.js

```diff
@@ -1543,16 +1543,16 @@ function initU2FAuth() {
             });
         }).catch(function () {
             // Fallback in case browser do not support U2F
-            window.location.href = "/user/two_factor"
+            window.location.href = suburl + "/user/two_factor"
```

Comment:
I just realized this isn't the URL anymore, it's `/user/settings/security`. This change will need to be made in master. Sorry for missing that in the original review.

Reply:
@techknowlogick this shuld be fixed on other PR then

Reply:
ok. sounds good.